lib/engine/game/g_18_eus/step/game_button_provider.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Engine
+  module Game
+    module G18EUS
+      module Step
+        module GameButtonProvider
```

Comment:
I see what you are going for with the name; however, it doesn't fit the codebase. Simply GameButtons is fine as a name.

Reply:
Changed